warehouse/query-core/src/main/java/datawave/query/jexl/functions/EvaluationPhaseFilterFunctions.java

```diff
@@ -1592,7 +1596,7 @@ static long getNextTime(long time, int granularity) {
      * @throws ParseException
      *             if the value failed to be parsed using the supplied format
      */
-    protected static long getTime(Object value, DateFormat format) throws ParseException {
+    static long getTime(Object value, DateFormat format) throws ParseException {
```

Comment:
It's a nitpick but I would prefer these be `public static...`

Reply:
Create pr/1368 branch in datawave repo to allow integration testing to start.